.typo-ci.yml

```diff
@@ -168,5 +168,9 @@ excluded_words:
   - regexes
   - walkthroughs
   - venv
+  - regexfeaturizer
+  - crfentityextractor
+  - Comerica
```

Comment:
🤔 what is this

Reply:
they were cited as typos because they were parts of links that it couldn't recognize. Comerica was just part of an example.